docker-compose.yml

```diff
@@ -18,4 +18,6 @@ services:
       - postgres
 
   concept-catalogue-gui:
-    image: dcatno/concept-catalogue-gui:latest
\ No newline at end of file
+    image: dcatno/concept-catalogue-gui:latest
+    expose:
+      - "8080"
```

Comment:
Will this port work with SSO?

Reply:
this is internal port, sso will not do local network requests to frontend server. The only time sso does server side request, is the token exchange against upstream idp (idplocal or idporten.mock)